<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>528</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MX Package Manager</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabApps">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Popular applications</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0" colspan="5">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="uniformRowHeights">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>false</bool>
         </property>
         <property name="headerHidden">
          <bool>false</bool>
         </property>
         <attribute name="headerCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="headerHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Package</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Info</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="searchPopular">
         <property name="placeholderText">
          <string>search</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Manage popular packages&lt;/span&gt;&lt;/p&gt;&lt;p&gt;To use this feature, make sure you are connected to the internet.&lt;/p&gt;&lt;p&gt;Greyed out items have already been installed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>132</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOtherRepos">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <attribute name="title">
       <string>Test and Backports Repos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>109</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;= Upgradable package. Newer version available in testing repository.</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="indent">
          <number>-1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="comboFilter">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Filter packages according to their status.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="currentText">
          <string>All packages</string>
         </property>
         <item>
          <property name="text">
           <string>All packages</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Installed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Upgradable</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Not installed</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="searchBox">
         <property name="placeholderText">
          <string>search</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Greyed out items have already been installed.</string>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <property name="indent">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <widget class="QTreeWidget" name="treeWidget_2">
         <column>
          <property name="text">
           <string notr="true"/>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Package Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Version</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Displayed</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Select source</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioMXtest">
            <property name="text">
             <string>MX Test Repo</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioBackports">
            <property name="text">
             <string>Debian Backports Repo</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="buttonBar" columnstretch="0,0,0,0,0,0,0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="0" column="6">
      <widget class="QPushButton" name="buttonUninstall">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Uninstall</string>
       </property>
       <property name="icon">
        <iconset theme="archive-remove">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonAbout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>About this application</string>
       </property>
       <property name="text">
        <string>About...</string>
       </property>
       <property name="icon">
        <iconset theme="help-about">
         <normaloff/>
        </iconset>
       </property>
       <property name="shortcut">
        <string>Alt+B</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonHelp">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Display help </string>
       </property>
       <property name="text">
        <string>Help</string>
       </property>
       <property name="icon">
        <iconset theme="help-browser">
         <normaloff/>
        </iconset>
       </property>
       <property name="shortcut">
        <string>Alt+H</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QPushButton" name="buttonInstall">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Install</string>
       </property>
       <property name="icon">
        <iconset theme="run-install">
         <normaloff/>
        </iconset>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <spacer name="horizontalSpacer2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="9">
      <widget class="QPushButton" name="buttonCancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Quit application</string>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
       <property name="icon">
        <iconset theme="window-close">
         <normaloff/>
        </iconset>
       </property>
       <property name="shortcut">
        <string>Alt+N</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelMX">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/icons/icons/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>searchPopular</tabstop>
  <tabstop>treeWidget</tabstop>
  <tabstop>buttonAbout</tabstop>
  <tabstop>buttonHelp</tabstop>
  <tabstop>buttonInstall</tabstop>
  <tabstop>buttonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>pressed()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>723</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
